mobile/examples/phi-3/ios/LocalLLM/LocalLLM/GenAIGenerator.mm

```diff
@@ -66,18 +80,17 @@ - (void)generate:(nonnull NSString*)input_user_question {
     NSLog(@"Setting generator parameters...");
     auto params = OgaGeneratorParams::Create(*self->model);
     params->SetSearchOption("max_length", kMaxTokens);
-    params->SetInputSequences(*sequences);
 
     auto generator = OgaGenerator::Create(*self->model, *params);
 
     bool isFirstToken = true;
     NSLog(@"Starting token generation loop...");
 
     startTime = Clock::now();
+    firstTokenTime = Clock::now();
+    generator->AppendTokenSequences(*sequences);
     while (!generator->IsDone()) {
       tokenStartTime = Clock::now();
-
```

Comment:
Planning to merge this branch when GenAI API changes are released in next update